Assignment2/juanked-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,65 @@
+{
+    "@context": {
+        "obj": "http://www.w3.org/2001/obj",

```

Comment:
Add / at the end of the URI

Reply:
Hi, I did add the / at the end of the URI (as you can see in my "change done" commit), I don't know what else should change to satisfy the change request

Reply:
Sorry, you are right. It is ok.